web/src/Web.App/Views/Shared/Components/Cookies/Default.cshtml

```diff
@@ -0,0 +1,86 @@
+<div class="govuk-cookie-banner" data-nosnippet role="region" aria-label="Cookies on @Constants.ServiceName" id="cookies-banner">
+    <div class="govuk-cookie-banner__message govuk-width-container" id="unselected-cookies-banner">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <h2 class="govuk-cookie-banner__heading govuk-heading-m">
+                    Cookies on @Constants.ServiceName
+                </h2>
+                <div class="govuk-cookie-banner__content">
+                    <p class="govuk-body">We use some essential cookies to make this service work.</p>
+                    <p class="govuk-body">We'd also like to use analytics cookies so we can understand how you use the service and make improvements.</p>
+                </div>
+            </div>
+        </div>
+        <div class="govuk-button-group">
+            <button type="button" class="govuk-button" data-module="govuk-button" id="accept-cookies">
+                Accept analytics cookies
+            </button>
+            <button type="button" class="govuk-button" data-module="govuk-button" id="reject-cookies">
+                Reject analytics cookies
+            </button>
+            <a class="govuk-link" href="@Url.Action("Cookies", "StaticContent")">View cookies</a>
+        </div>
+    </div>
+    <div class="govuk-cookie-banner__message govuk-width-container" role="alert" hidden id="accepted-cookies-banner">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <div class="govuk-cookie-banner__content">
+                    <p class="govuk-body">You've accepted analytics cookies. You can <a class="govuk-link" href="@Url.Action("Cookies", "StaticContent")">change your cookie settings</a> at any time.</p>
+                </div>
+            </div>
+        </div>
+        <div class="govuk-button-group">
+            <button type="button" class="govuk-button hide-cookies" data-module="govuk-button">
+                Hide cookie message
+            </button>
+        </div>
+    </div>
+    <div class="govuk-cookie-banner__message govuk-width-container" role="alert" hidden id="rejected-cookies-banner">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <div class="govuk-cookie-banner__content">
+                    <p class="govuk-body">You've rejected analytics cookies. You can <a class="govuk-link" href="@Url.Action("Cookies", "StaticContent")">change your cookie settings</a> at any time.</p>
+                </div>
+            </div>
+        </div>
+        <div class="govuk-button-group">
+            <button type="button" class="govuk-button hide-cookies" data-module="govuk-button">
+                Hide cookie message
+            </button>
+        </div>
+    </div>
+</div>
+
+<script type="text/javascript" add-nonce="true">
```

Comment:
Can this be added to the script section?

Reply:
Wrapping in `@section scripts` doesn't seem to render anything at all from a component view